Jenkinsfile

```diff
@@ -46,6 +46,13 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+    if(env.WORKSPACE.toString() == 'null') {
```

Comment:
`env.WORKSPACE` seems to be a piece of magic, `!env.WORKSPACE` and
`env.WORKSPACE == null` do not work.  This is the closest that works.